tests/drizzle/test_cdriz.py

```diff
@@ -0,0 +1,62 @@
+import pytest
+import numpy as np
+import cdriz_setup
+
+
+@pytest.fixture
+def kernel_pars():
+    # get cdriz.triz inputs from cdriz_setup.py
+    _params = cdriz_setup.Get_Grid(inx=10, iny=10, outx=13, outy=13)
+    _params.zero_background()
+    return _params
+
+
+# "square", "point", "turbo", "gaussian", "lanczos3"
+@pytest.mark.parametrize("kernel", ["square", "point", "turbo"])
+def test_square_kernel(kernel_pars, kernel, return_png=False):
+    """Function tests different c code point kernels (inputs already created on instantiation).
+
+    Parameters
+    ----------
+    kernel_pars : Class
+        The Class inintialized in Get_Grid which includes all of the inputs need to run cdriz.tdriz.
+    kernel: str [argument passed with parameterize]
+        The name of the kernel being used.
+    return_png:
+        Whether to return a png of the outputs.
+    """
+    # add bright pixel at center
+    kernel_pars.insci[3, 3] = 1e4
+
+    # resample:
+    cdriz_setup.cdriz_call(kernel_pars, kernel)
+
+    if return_png:
+        # save truth file as figure
+        cdriz_setup.generate_png(
+            kernel_pars, "./tests/drizzle/truth_files/cdriz_square.png"
```

Comment:
I had not expected for this much of a path to the file to still be needed.  I thought something more like "./truth_files/cdriz_square.png" would be needed to append to the os.path.dirname(dunderscorefiledunderscore)? If I am wrong, please ignore this comment for all the locations where generate_png() is invoked.

Reply:
I've tried using the __file__ and it ends up putting the pngs in the main module directory.